djstripe/stripe_objects.py

```diff
@@ -50,8 +51,12 @@
 
 @python_2_unicode_compatible
 class StripeObject(TimeStampedModel):
-    # This must be defined in descendants of this model/mixin
-    # e.g. "Event", "Charge", "Customer", etc.
+    """
```

Comment:
These were comments on the next two fields. The docstring should say something about being the base class for all stripe objects and about holding the core functionality that they all use